lib/steps/create_and_checkout_branch_step.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 type CreateAndCheckoutBranchStep struct {
+	NoAutomaticAbort
```

Comment:
There is a method called `ShouldAutomaticallyAbortOnError`, so I would expect this to be named `NoAutomaticAbortOnError`.